go/ct/driver/cli/utils.go

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by the GNU Lesser General Public Licence v3
+//
+
+package cliUtils
+
+import (
+	"regexp"
+	"runtime"
+
+	"github.com/urfave/cli/v2"
+)
+
+type filterFlagType struct {
```

Comment:
would it make sense to add also a `FlagInterface` with `GetFlag`, `Fetch` and `flag T` ?

Reply:
Fetch function may have different types. 
I wanted to remove the regex parsing code from the caller side.